### ISSUE TITLE
Update to later versions of Scalacheck

### ISSUE THREAD

**Issue by nrinaudo** [nrinaudo/kantan.codecs#15]
`Cogen` is both great and a pain to upgrade to.

**Comment by nrinaudo** [comment]
This is unfortunately currently impossible: @typelevel / cats' laws are only built for scalacheck `1.12.5` and incompatible with `1.3.0+`.

For a concrete example of what happens:
```
[info]   java.lang.NoSuchMethodError: org.scalacheck.Arbitrary$.arbFunction1(Lorg/scalacheck/Arbitrary;)Lorg/scalacheck/Arbitrary;
[info]   at cats.laws.discipline.MonadTests$$anon$2.props(MonadTests.scala:30)
[info]   at org.typelevel.discipline.Laws$RuleSet$class.org$typelevel$discipline$Laws$RuleSet$$collectParentProps(Laws.scala:80)
[info]   at org.typelevel.discipline.Laws$RuleSet$$anonfun$all$1.apply(Laws.scala:83)
[info]   at org.typelevel.discipline.Laws$RuleSet$$anonfun$all$1.apply(Laws.scala:83)
[info]   at org.typelevel.discipline.AllProperties.<init>(Laws.scala:128)
[info]   at org.typelevel.discipline.Laws$RuleSet$class.all(Laws.scala:83)
[info]   at cats.laws.discipline.MonadTests$$anon$2.all(MonadTests.scala:25)
[info]   at org.typelevel.discipline.scalatest.Discipline$class.checkAll(Discipline.scala:12)
```

**Comment by nrinaudo** [comment]
It looks like cats' issues are already being fixed: typelevel/cats#1345.
